package.json

```diff
@@ -27,8 +32,24 @@
     "chai-as-promised": "^7.1.1",
     "chai-datetime": "^1.5.0",
     "coffeescript": "^2.3.2",
-    "mocha": "^5.2.0",
+    "husky": "^1.3.1",
+    "lint-staged": "^8.1.5",
+    "mocha": "^6.0.2",
+    "prettier": "^1.16.4",
```

Comment:
NIT: prettier is already installed by resin-lint

Reply:
Yeah but we also use it directly so I want to be explicit about it being a dependency